justfile

```diff
@@ -106,7 +106,19 @@ build-docker-images:
 # generate rust bindings for contracts
 REGEXP := "^LightClient$|^LightClientArbitrum$|^LightClientStateUpdateVK$|^FeeContract$|PlonkVerifier$|^ERC1967Proxy$|^LightClientMock$|^LightClientStateUpdateVKMock$|^PlonkVerifier2$|^PermissionedStakeTable$"
 gen-bindings:
-    forge bind --contracts ./contracts/src/ --crate-name contract-bindings --bindings-path contract-bindings --select "{{REGEXP}}" --overwrite --force
+    # Update the git submodules
+    git submodule update --init --recursive
+
+    # Generate the ethers bindings
+    forge bind --contracts ./contracts/src/ --ethers --crate-name contract-bindings-ethers --bindings-path contract-bindings-ethers --select "{{REGEXP}}" --overwrite --force
+
+    # Generate the alloy bindings
+    # TODO: `forge bind --alloy ...` fails if there's an unliked library but what are the implications of setting the address here?
+    forge bind --skip test --skip script --libraries contracts/src/libraries/PlonkVerifier.sol:PlonkVerifier:0x5fbdb2315678afecb367f032d93f642f64180aa3 --alloy --contracts ./contracts/src/ --crate-name contract-bindings-alloy --bindings-path contract-bindings-alloy --select "{{REGEXP}}" --overwrite --force
```

Comment:
The linking done here feels very wrong because I believe we may end up deploying contracts pointing to contract address in the development environment, even if we are in other environments. So still have to check what to do about this.

Reply:
Sort of fixed here: https://github.com/EspressoSystems/espresso-sequencer/pull/2456/commits/6b41996faa6e5a9a70c434ecf17f70aecf0512a6